Docs/ContinuousIntegration.md

```diff
@@ -22,9 +22,9 @@ Normally when using a code formatter like CSharpier, you'll want to ensure that
 2. Use your preferred CI/CD tool to run the following commands.
    ```bash
    dotnet tool restore
-   dotnet-csharpier --check
+   dotnet tool run dotnet-csharpier --check .
```

Comment:
Both `dotnet csharpier` and `dotnet-csharpier` work locally for me and in the workflows I have set up. I'm running windows, and the workflows run on ubuntu.
Most places I'm looking don't mention `dotnet tool run` at all, besides the microsoft doc for creating a tool.
The docs for these don't mention `dotnet tool run`
https://www.nuget.org/packages/dotnet-ef/
https://www.nuget.org/packages/dotnet-serve/

Could this be a problem specific to your setup somehow?

The `.` definitely needs to be there though.

Reply:
Here is an example I actually tried.
On ubuntu-latest, the `dotnet-csharpier` command does not work after restore. `dotnet tool run` works reliably.
https://github.com/eai04191/actions-testing/commits/csharpier

Reply:
I see from your commit log you've tried `dotnet-csharpier` and it didn't work.
Could you please also try
```
dotnet csharpier --check .
```

Reply:
I'm late to fix but I fixed it like that so I think we can merge.